internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -13,10 +14,35 @@ import (
 	"github.com/aws/copilot-cli/internal/pkg/template"
 )
 
+var (
+	pEnabled  = aws.String("ENABLED")
+	pDisabled = aws.String("DISABLED")
+)
+
+// Default values for EFS options
+var (
+	defaultRootDirectory   = aws.String("/")
+	defaultIAM             = pDisabled
+	defaultReadOnly        = aws.Bool(true)
+	defaultWritePermission = false
```

Comment:
It might not be very safe to define const values as vars here, since they might get accidentally changed.